<launch>
<!--<node pkg = "pointcloud_filter" type= "pointcloud_filter_passthrough" name="trim_height" output="screen">
    <remap from="input" to="/rslidar_points" />
</node>-->

<node pkg = "pointcloud_filter" type="pointcloud_filter_road" name="filter_road" output="screen" >
<remap from="cloud_input" to="/rslidar_points" />
  </node>

<node pkg="pointcloud_to_laserscan" type="pointcloud_to_laserscan_node" name="pointcloud_to_laserscan">
<remap from="cloud_in" to="/object_on_road"/>
<remap from="scan" to="/topdown_scan"/>
<rosparam>
transform_tolerance: 0.01
min_height: 0.0
max_height: 0.5

angle_min: -3.141569 # -M_PI/2
angle_max: 3.141569 # M_PI/2
angle_increment: 3.141569/360 # M_PI/360.0
scan_time: 0.01
range_min: 0.40
range_max: 100.0
use_inf: false

# Concurrency level, affects number of pointclouds queued for processing and number of threads used
# 0 : Detect number of cores
# 1 : Single threaded
# 2->inf : Parallelism level
concurrency_level: 1
</rosparam>
</node>


</launch>
